listenbrainz/db/tests/test_recommendations_cf_recording_feedback.py

```diff
@@ -112,54 +111,99 @@ def test_get_feedback_for_user(self):
         self.assertEqual(result[0].recording_mbid, self.sample_feedback[0]["recording_mbid"])
         self.assertEqual(result[0].rating, self.sample_feedback[0]["rating"])
 
+        feedback_love = []
         for i in range(60):
+            rec_mbid = str(uuid.uuid4())
             db_feedback.insert(
                 RecommendationFeedbackSubmit(
                     user_id=self.user2['id'],
-                    recording_mbid=str(uuid.uuid4()),
+                    recording_mbid=rec_mbid,
+                    rating='love'
+                )
+            )
+            # prepended to the list since ``get_feedback_for_users`` returns data in descending
+            # order of creation.
+            feedback_love.insert(
```

Comment:
Is there a reason why you save the mbid and create the object twice? Why not create the object once and save it for adding to the feedback_love[] ?

Reply:
No, I don't know why I did that 🙄. 
Pushing the changes.